dbt/adapters/redshift/impl.py

```diff
@@ -114,6 +115,44 @@ def valid_incremental_strategies(self):
     def timestamp_add_sql(self, add_to: str, number: int = 1, interval: str = "hour") -> str:
         return f"{add_to} + interval '{number} {interval}'"
 
+    def _get_cursor(self):
+        return self.connections.get_thread_connection().handle.cursor()
+
+    def _get_tables(self, database: Optional[str], schema: Optional[str]):
+        cursor = self._get_cursor()
+        results = []
+        for table in cursor.get_tables(
+            catalog=database,
+            schema_pattern=schema,
+            table_name_pattern=None,
+            types=["VIEW", "TABLE"],
+        ):
+            results.append([table[0], table[1], table[2], table[3]])
+            # kinda feel like it might not be a good thing to do this by index
+        return results
+
+    def list_relations_without_caching(  # type: ignore
+        self, schema_relation: BaseRelation
+    ) -> List[RedshiftRelation]:
+        results = self._get_tables(schema_relation.database, schema_relation.schema)
+        relations = []
+        quote_policy = {"database": True, "schema": True, "identifier": True}
```

Comment:
@mikealfare Not super familiar with QuotePolicy - this part of the code is from the sqladapter [here](https://github.com/dbt-labs/dbt-core/blob/f9abeca231ad20a8d0c4d8154b60bfda48d7fef5/core/dbt/adapters/sql/impl.py#L175)

Reply:
I'm not super familiar with this either but my understanding is that `QuotePolicy` should be based on the adapter (i.e. snowflake/spark set their own because they differ from the postgres based default).